scripts/run_finnemit.py

```diff
@@ -0,0 +1,8 @@
+import finnemit
```

Comment:
Did you mean to commit this file? And if so, do we need to have it in the finnemit library?

It looks like it's essentially wrapping the `get_emissions` function for command line use. Maybe you were using it while debugging?